source/mapping-rules/new-rules-suggested-mappings.txt

```diff
@@ -58,6 +58,10 @@ Steps
 
 #. Deselect the suggested mapping rules you do not want applied to your MongoDB schema.
 
+   If you do want to use any of the suggested mapping rules, click the :guilabel:`Ignore` 
```

Comment:
Should be "If you do not", right?
I'd also drop the "to minimize the :guilabel:`Suggested 
   Mappings` pane." text as this could change in the future.

Reply:
Thank you, updated :)